spec/student_spec.rb

```diff
@@ -25,11 +28,12 @@
 
     it 'has an id that defaults to `nil` on initialization' do
       expect(josh.id).to eq(nil)
+      DB[:conn].execute("DROP TABLE students")
```

Comment:
Ah, that makes perfect sense. Yeah, this change still solves the issue we were having. I will do a quick commit to update.